core/src/voting_service.rs

```diff
@@ -90,20 +90,26 @@ impl VotingService {
 
         let mut measure = Measure::start("vote_tx_send-ms");
         let target_address = target_address.unwrap_or_else(|| cluster_info.my_contact_info().tpu);
-        let _ = get_connection(&target_address).send_transaction(vote_op.tx());
+        let _ = get_connection(&target_address).serialize_and_send_transaction(vote_op.tx());
```

Comment:
I think it's better to keep vote transactions as `Transaction` for now to avoid the `into_legacy_transaction` unwrap. Can you manually serialize and use `send_wire_transaction` directly here?

Reply:
I'll follow up with a change, thanks for fixing CI